apps/labs/posts/integrating-hypothesis-into-sympy.md

```diff
@@ -0,0 +1,97 @@
+---
+title: 'Integrating Hypothesis into SymPy'
+authors: [diane-tc]
+published: September 20, 2023
+description: 'Gives an introduction to the utility of hypothesis in SymPy'
+category: [Property Based Testing]
+featuredImage:
+  src: /posts/integrating-hypothesis-into-sympy/blog_feature_org.png
+  alt: 'An illustration of a brown hand holding up a microphone, with some graphical elements highlighting the top of the microphone.'
+hero:
+  imageSrc: /posts/integrating-hypothesis-into-sympy/blog_hero_var2.svg
+  imageAlt: 'An illustration of a brown hand holding up a microphone, with some graphical elements highlighting the top of the microphone.' 
+---
+This summer I interned at Quansight Labs with a focus of intergrating [Hypothesis](https://github.com/HypothesisWorks/hypothesis/) into the testing suit of [SymPy](https://github.com/sympy/sympy). The primary [pull request to add Hypothesis to the test infrastructure](https://github.com/sympy/sympy/pull/25428) to complete this was simple. The primary challenges lied thereafter: questions around the _utility of hypothesis_ and its appropriate usage arose. 
```

Comment:
```suggestion
This summer I interned at Quansight Labs with a focus of integrating [Hypothesis](https://github.com/HypothesisWorks/hypothesis/) into the testing suit of [SymPy](https://github.com/sympy/sympy). The primary [pull request to add Hypothesis to the test infrastructure](https://github.com/sympy/sympy/pull/25428) to complete this was simple. The primary challenges lied thereafter: questions around the _utility of hypothesis_ and its appropriate usage arose. 
```
It also may make sense to remove one of the `primary` instances just to avoid repetition.